fixtures/cameo/root-par-6.json

```
@@ -0,0 +1,761 @@
+        {
+          "dmxRange": [46, 53],
+          "type": "ColorPreset",
+          "comment": "Turquoise"
+        },
+        {
+          "dmxRange": [54, 61],
+          "type": "ColorPreset",
+          "comment": "Cyan"
+        },
+        {
+          "dmxRange": [62, 69],
+          "type": "ColorPreset",
+          "comment": "Blue"
+        },
+        {
+          "dmxRange": [70, 77],
+          "type": "ColorPreset",
+          "comment": "Lavender"
+        },
+        {
+          "dmxRange": [78, 85],
+          "type": "ColorPreset",
+          "comment": "Mauve"
+        },
+        {
+          "dmxRange": [86, 93],
+          "type": "ColorPreset",
+          "comment": "Magenta"
+        },
+        {
+          "dmxRange": [94, 101],
+          "type": "ColorPreset",
+          "comment": "Pink"
+        },
+        {
+          "dmxRange": [102, 109],
+          "type": "ColorPreset",
+          "comment": "Warm White"
+        },
+        {
+          "dmxRange": [110, 117],
+          "type": "ColorPreset",
+          "comment": "White"
+        },
+        {
+          "dmxRange": [118, 125],
+          "type": "ColorPreset",
+          "comment": "Cold White"
+        },
+        {
+          "dmxRange": [126, 127],
+          "type": "Effect",
+          "effectPreset": "ColorJump",
+          "speed": "stop",
+          "comment": "Color Jumping Stop"
+        },
+        {
+          "dmxRange": [128, 164],
+          "type": "Effect",
+          "effectPreset": "ColorJump",
+          "speedStart": "slow",
+          "speedEnd": "fast",
+          "comment": "Color Jumping speed slow -> fast / Color  1 -> 12"
+        },
+        {
+          "dmxRange": [165, 201],
+          "type": "Effect",
+          "effectPreset": "ColorFade",
+          "speedStart": "slow",
+          "speedEnd": "fast",
+          "comment": "Color Fading speed slow -> fast / Color  1 -> 12"
+        },
+        {
+          "dmxRange": [202, 207],
+          "type": "ColorPreset",
+          "comment": "User Color_1"
+        },
+        {
+          "dmxRange": [208, 213],
+          "type": "ColorPreset",
+          "comment": "User Color_2"
+        },
+        {
+          "dmxRange": [214, 219],
+          "type": "ColorPreset",
+          "comment": "User Color_3"
+        },
+        {
+          "dmxRange": [220, 225],
+          "type": "ColorPreset",
+          "comment": "User Color_4"
+        },
+        {
+          "dmxRange": [226, 255],
+          "type": "ColorPreset",
+          "comment": "UV"
+        }
+      ]
+    },
+    "DMX Delay": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 5],
+          "type": "EffectParameter",
+          "parameter": "off",
+          "comment": "No Delay"
+        },
+        {
+          "dmxRange": [6, 255],
+          "type": "EffectParameter",
+          "parameterStart": 0.1,
+          "parameterEnd": 2,
+          "comment": "Delay in seconds"
+        }
```

These should be `Maintenance` capabilities, as the DMX delay is likely to only be calibrated once per venue and not changed all the time like colors/brightness etc.

```suggestion
    "DMX Delay": {
      "capabilities": [
        {
          "dmxRange": [0, 5],
          "type": "Maintenance",
          "parameter": "off",
          "comment": "No Delay"
        },
        {
          "dmxRange": [6, 255],
          "type": "Maintenance",
          "parameterStart": 0.1,
          "parameterEnd": 2,
          "comment": "Delay in seconds"
        }
```